libyul/optimiser/ForLoopConditionOutOfBody.cpp

```diff
@@ -53,7 +53,7 @@ void ForLoopConditionOutOfBody::operator()(ForLoop& _forLoop)
 	if (!SideEffectsCollector(m_dialect, *firstStatement.condition).movable())
 		return;
 
-	YulName iszero = m_dialect.booleanNegationFunction()->name;
+	YulName const iszero {m_dialect.builtin(*m_dialect.booleanNegationFunctionHandle()).name};
```

Comment:
```suggestion
	YulName const iszero = m_dialect.builtin(*m_dialect.booleanNegationFunctionHandle()).name;
```

Reply:
I see you changed it too `= YulName{}`. I guess the constructor is `explicit`? I thought that was not the case. In that case the original was actually fine (just without the unnecessary space).

Reply:
Yup, `explicit` indeed. Changed it back (sans space).